torch/csrc/Module.cpp

```diff
@@ -442,7 +443,7 @@ static PyObject* initModule() {
   ASSERT_TRUE(THPWrapper_init(module));
   ASSERT_TRUE(THPGenerator_init(module));
   ASSERT_TRUE(THPException_init(module));
-  ASSERT_TRUE(THPSize_init(module));
+  THPSize_init(module);
```

Comment:
Why'd you change this?  Seems nicer to just have everything return a boolean for consistency, even if they can't be false.

Reply:
Nothing should return a boolean, but I'm too lazy to clean-up everything at once.

The whole `ASSERT_TRUE` thing ends up replacing useful error messages with worthless ones.